### PR TITLE
#164798216  User should be able to know reading time of an article

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

- Enables users  to know the reading time of an article

#### Description of Task to be completed?
-  When a user makes arequest for an article or articles, it should be having an estimated reading time for it

#### How should this be manually tested?

- Fetch this branch `git fetch origin ft-time-to-read-article`
- Checkout to the branch `git checkout ft-time-to-read-article`
- Run `python manage.py runserver`
- Access get all articles endpoint `{{base-url}}/api/articles/`

#### What are the relevant pivotal tracker stories?
[#164798216](https://www.pivotaltracker.com/story/show/164798216)

#### Postman Collection
[Run in Postman](https://app.getpostman.com/run-collection/c6f646a632be8b0634ee)

### CODE REVIEW

**Review by ezrogha — DISMISSED** [review]
LGTM

File: .env_config_sample [hunk 1]
```diff
@@ -8,4 +8,7 @@ SOCIAL_USER_PASS = 'SpartaAuthorsHaven2019'
 EMAIL_HOST_USER = 'your email'
 EMAIL_HOST_PASSWORD = 'your password'
 EMAIL_HOST='smtp.gmail.com'
-EMAIL_PORT='587'
+EMAIL_PORT='587
```
Comment (dannylwe) [inline .env_config_sample]: @felixkiryowa kindly avoid introducing unnecessary. Kindly restore the missing quotation mark on line 11

**Review by dorothyas — DISMISSED** [review]

File: authors/apps/articles/models.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
+from django.contrib.postgres import fields
 from django.db import models
-from django.utils import timezone
-from authors.apps.authentication.models import User
```
Comment (dannylwe) [inline authors/apps/articles/models.py:3]: @felixkiryowa kindly stop making unnecessary changes, for example, the User model. This may introduce bugs. Restore changes.

**Review by dannylwe — CHANGES_REQUESTED** [review]

File: authors/apps/articles/models.py (hunk 2 above)
Comment (felixkiryowa) [reply authors/apps/articles/models.py:3]: I really don't understand this because I did not change anything in respect to those imports 
I only added these imports
`from django.conf import settings`
`import math`

Comment (felixkiryowa) [reply authors/apps/articles/models.py:3]: If not they were unused imports, so there was no need for them to be in that file

**Review by dannylwe — APPROVED** [review]

**Review by ezrogha — APPROVED** [review]

**Review by ja-odur — APPROVED** [review]
LGTM
